gem/lib/phlexing/template_generator.rb

```diff
@@ -34,7 +34,7 @@ def handle_text_output(text)
     end
 
     def handle_html_comment_output(text)
-      output("comment", quote(text))
+      output("comment", block { quote(text) })
```

Comment:
@willcosgrove I think this should do it:
```suggestion
  output("comment", "{ #{quote(text)} }")
```

Reply:
Oh, my bad. I just saw you wrote that here too: https://github.com/marcoroth/phlexing/pull/100#issuecomment-1432034416

Otherwise we could also introduce another helper in [`gem/lib/phlexing/helpers.rb`](https://github.com/marcoroth/phlexing/blob/96abde449a0caecd5e7251e56f296d0e7be26af0/gem/lib/phlexing/helpers.rb), like:

```ruby
    def braces(string)
      "{ #{string} }"
    end
```

And then call it like:
```suggestion
      output("comment", braces(quote(text)))
```